R/utilities.R

```diff
@@ -1077,3 +1077,65 @@ release2json <- function(release, nameslist){
     return(result)
 
 }
+
+#' Error check imputation range for numeric or integer variables
+#' 
+#' Check that the entered imputation range is within the entered data range. If yes, return
```

Comment:
Add statement that emphasizes that it is a privacy concern if imputation range is not within entered data range